etc/config/csharp.win32.properties

```diff
@@ -0,0 +1,12 @@
+# environment variables needed to run dotnet and crossgen
+
+envVars=LOCALAPPDATA=C:\Users\Jason\AppData\Local;APPDATA=C:\Users\Jason\AppData\Roaming;TEMP=C:\Users\Jason\AppData\Local\Temp;USERPROFILE=C:\Users\Jason;ProgramFiles=C:\Program Files;CoreCLRDebugPath=C:\GitHub\coreclr\bin\Product\Windows_NT.x64.Debug
```

Comment:
@partouf here's an example of needing `:` in an environment variable.

@jasonmeisel you probably don't want to hardcode paths to your username here. I'd suggest you put these in a (non-checked in) `csharp.local.properties`. Though leaving a comment here explaining what environment variables are needed might make sense too.

Reply:
Added a comment with a list of necessary environment variables, replaced "Jason" with "<User>"